web/src/components/confirm-form.jsx

```diff
@@ -71,14 +72,22 @@ class ConfirmForm extends React.Component {
         )}
 
         <section id="confirm-buttons">
+
+          { pendingSentences || readyCount === 0 ? 
```

Comment:
`readyCount` is used to indicate that no sentence is ready for submission (for example if all added sentences are invalid - one way to test this is using only sentences with numbers in it). In that case, we only want to have a disabled button, not a spinning button.

Reply:
<img width="862" alt="screenshot 2019-02-24 at 12 11 52" src="https://user-images.githubusercontent.com/330324/53298377-6d4da680-382d-11e9-8bc6-66f6d62e4bdc.png">

Reply:
Ok, done & tested :)